torch/distributed/examples/memory_tracker_example.py

```diff
@@ -0,0 +1,33 @@
+import torch
+import torchvision
+
+from torch.distributed._tools import MemoryTracker
+
+
+def run_one_model(net, input):
```

Comment:
nit: Shall we add type annotations for this? (mainly for `input: torch.Tensor` since inputs may sometimes be some data structure)

Reply:
sounds good